### DISCUSSION TITLE
2023-09-06

### DISCUSSION THREAD

**Discussion by jaehee329** [woowacourse-teams/2023-haru-study#478] [daily_meeting]
### 오늘의 할 일
백: 오전 강의
개인 미션 진행(프 제출 목)

### 내일 할 일
사용자 유치 계획 discussion에 하나씩 아이디어 남기기 (9/8)
개인용 서비스를 유지하는 방향에 대해 논의

**Comment by nlom0218** [comment]
### 체크인

9점 - 이렇게 평화로워도 되는 건가요...? 😇

### 체크아웃

8점 - 성능 최적화가 뭔가 찝찝하네욤..

**Comment by aak2075** [comment]
### 체크인

10점 - 오늘 하루도 화이팅

### 체크아웃

10점 - atomic요거 맛있네요

**Comment by woosung1223** [comment]
### 체크인
7점 - 할게 많아서 슬픕니다

### 체크아웃
7점 - 낫 배드

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 아침에 톰캣 좀 공부해보려 했는데 이거 쉽지 않네요 ㅋㅋㅋ...

### 체크아웃
6점 - 여전히 톰캣은 어렵고 스레드도 해야되는데 허허... 미션도 해야되는데... 프로젝트도 해야되는데... 정신 똑바로 잡아야 할 레벨4네요...

**Comment by jaehee329** [comment]
### 체크인
3점 - 점점 피곤해지는..
### 체크아웃
2점 - 조금 더 피곤해지는..

**Comment by yeopto** [comment]
### 체크인

10점 - 피곤하지만 좋네요.

### 체크아웃

10점 - 미션 제출해서 한시름 덜었네용

**Comment by woo-jk** [comment]
### 체크인
7점 - 아침 피곤함이 안풀린다,,

### 체크아웃
8점 - 미션 끝나서 기분은 좋지만 아직 할게 많이 남아있따
